android/app/src/main/java/com/rntemplate/MainActivity.java

```diff
@@ -4,12 +4,12 @@
 
 public class MainActivity extends ReactActivity {
 
-    /**
-     * Returns the name of the main component registered from JavaScript.
-     * This is used to schedule rendering of the component.
-     */
-    @Override
-    protected String getMainComponentName() {
-        return "rntemplate";
-    }
+  /**
+   * Returns the name of the main component registered from JavaScript. This is used to schedule
+   * rendering of the component.
```

Comment:
....the rendering of...

Reply:
oh, we can open a pull request in react native itself 😄